packages/ui-extensions/src/surfaces/customer-account/api.ts

```diff
@@ -40,10 +51,6 @@ export type {
   NumberRange,
   Order,
   OrderStatusApi,
-  OrderStatusCompany,
-  OrderStatusLocalization,
-  OrderStatusPurchasingCompany,
-  OrderStatusBuyerIdentity,
```

Comment:
Same with these types, won't that cause us another breaking type change when we need to move the types back into stable ?

Reply:
Likely, but it will be in `unstable` / the next major version and at least it won't be breaking for people who switch from OSP to ODP now, right?